commitlint.config.js

```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'type-enum': [2, 'always', ['fix', 'feat']],
+    'scope-empty': [0],
+    'header-max-length': [0],
+    'subject-case': [2, 'always', 'sentence-case'],
```

Comment:
Any chance of disabling this one too? This particular is very annoying to handle manually, but can be easily automated